### ISSUE TITLE
Sometimes all of the counters show 0 edits

### ISSUE THREAD

**Issue by Wesalius** [edsu/wikipulse#14]
Is it due to heroku limits or...?

**Comment by edsu** [comment]
Yes Heroku puts the webapp to sleep since I am not paying them. When it wakes up it needs to reconnect to the IRC channels where it gets updates. If it's important maybe we could run it in the wikimedia tools area so it can always be on?

**Comment by Wesalius** [comment]
Yes, I think runnning the whole thing in tool labs would be a good solution.
